frame/assets-freezer/Cargo.toml

```
@@ -0,0 +1,51 @@
+[package]
+name = "pallet-assets-freezer"
+version = "3.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "Extension pallet for managing frozen assets"
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+serde = { version = "1.0.101", optional = true }
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
+sp-std = { version = "3.0.0", default-features = false, path = "../../primitives/std" }
+# Needed for various traits. In our case, `OnFinalize`.
+sp-runtime = { version = "3.0.0", default-features = false, path = "../../primitives/runtime" }
+# Needed for type-safe access to storage DB.
+frame-support = { version = "3.0.0", default-features = false, path = "../support" }
+# `system` module provides us with all sorts of useful stuff and macros depend on it being around.
```

```suggestion
sp-runtime = { version = "3.0.0", default-features = false, path = "../../primitives/runtime" }
frame-support = { version = "3.0.0", default-features = false, path = "../support" }
```

reminiscence of `pallet-example` everywhere :D